public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -360,6 +360,9 @@ export class PanelModel implements DataConfigSource, IPanelModel {
   }
 
   runAllPanelQueries({ dashboardUID, dashboardTimezone, timeData, width }: RunPanelQueryOptions) {
+    if (this.type === 'row') {
+      return;
+    }
```

Comment:
@juanicabanas this should not be possible, what calls runAllQueries on a row model?? 

PanelStateWrapper is not for row models, so confused

Reply:
I'll take a deep look and understand what's happening

Reply:
@torkelo @AgnesToulet image-renderer-service is navigating to `/d-solo` page, which shows panel per panel thanks to `SoloPangelPage` component. 
Instead of rendering <DashboardRow /> if `panel.type === 'row'` or <DashboardPanel />, `SoloPanelPage` always renders <DashboardPanel /> https://github.com/grafana/grafana/blob/main/public/app/features/dashboard/containers/SoloPanelPage.tsx
I made a quick change to render <DashboardRow /> but I'm not sure if we should show the arrow, config and trash icons 

| As it is now   | With the change |
| ------------- | ------------- |
| <img width="1510" alt="image" src="https://github.com/grafana/grafana/assets/11707172/d1d3c7dd-8da7-4d04-88ea-aea6e324bc34">  | <img width="1512" alt="image" src="https://github.com/grafana/grafana/assets/11707172/52aa9568-a3c7-433b-ba9b-9de04fe62c9d"> |

Reply:
it's because it's a collapsed row, do we want collapsed rows in the report?

Reply:
Yes, the PDF export contains the dashboard with all rows un-collapsed.